applications/ConvectionDiffusionApplication/tests/convection_diffusion_test_factory.py

```diff
@@ -79,4 +79,8 @@ class BasicDiffusionStationaryTest(ConvectionDiffusionTestFactory):
 
 @KratosUnittest.skipIfApplicationsNotAvailable("StructuralMechanicsApplication")
 class SimpleThermoMechanicalTest(ConvectionDiffusionTestFactory):
-    file_name = "thermo_mechanical_tests/coupled_problem_test"
\ No newline at end of file
+    file_name = "thermo_mechanical_tests/coupled_problem_test"
+
+@KratosUnittest.skipIfApplicationsNotAvailable("StructuralMechanicsApplication")
+class SimpleThermoMechanicalTableAccessorTest(ConvectionDiffusionTestFactory):
+    file_name = "thermo_mechanical_tests/thermo_mech_accessor/coupled_accessor_test"
```

Comment:
```suggestion
    file_name = "thermo_mechanical_tests/thermo_mechanical_accessor/coupled_accessor_test"
```
Besides I don't think the subfolder is required at all.